Gemfile

```diff
@@ -78,6 +79,7 @@ group :test do
 end
 
 group :development do
+  gem 'letter_opener'
```

Comment:
It looks like one of these additions has also [broken the build](https://travis-ci.org/openaustralia/planningalerts/builds/223779317) so hopefully fixing this up with fix the build too :sparkles: